### ISSUE TITLE
Template: reorder the headers

### ISSUE THREAD

**Issue by MirjamvT** [publiccodenet/standard#135]
suggestion: reorder the headers to:

a.       What this does                      [or: the why]

b.       Measure                                [or: the how]

c.       What this does not do          [additional info re form/how]

d.       How to test                           [additional info re form/how - maybe rephrase to make more clear what is to be tested and for what?]

e.       What to do                           [or: the what, and the whom]

                     with subheaders for various groups 

Looking at this, we seem to miss info on the ‘when’?

**Comment by ericherman** [comment]
It seems the current ordering is slightly different than it had been, but does not fully reflect this suggestion.

If a new ordering is proposed, we should close this and open a new issue describing the envisioned change, otherwise let's plan to close this by end of May 2021.

**Comment by ericherman** [comment]
The current ordering is good enough for now.
